protostar/cheatable_starknet/cheatables/cheatable_entry_point.py

```diff
@@ -55,37 +55,27 @@
 
 @dataclass(frozen=True)
 class CheatableExecuteEntryPoint(ExecuteEntryPoint):
-    cheaters: "CairoCheaters"
     max_steps: Optional[int] = None
     "``None`` means default Cairo value."
 
     @classmethod
-    def create_with_cheaters(
+    def create_for_protostar(
```

Comment:
this method can be removed as well

Reply:
This method actually has some purpose. Callers don't have to specify every `ExecuteEntryPoint` constructor parameter — sensible default are used here. Also, this method can use `Address` and `Selector` types to avoid converting every time `ExecuteEntryPoint` is created.

Reply:
If you feel like it's useful - you can leave it, but the same defaults are in `create`. Please reuse that method then in this one